end_to_end/gpu/test_decode_gpu.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This script provides a convenient set of default configs used for testing decode.py on GPU
+
```

Comment:
How will `test_decode_gpu.sh` and `test_train_gpu.sh` be used? They seem kind of weird but I can see why they'd be useful as well!

Reply:
They make the test commands more concise by providing common default values for the testing.

They will be used like in [this line](https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/283/files#diff-495d4687b5bff3f6b5a39a3b16c59222e8f638729e635569a8e7d8ea93d351f9R48)
I think they can be also used in our unit tests.

Reply:
Agreed the files seem kind of weird at first, but your explanation makes sense. If you want to re-use the same set of config options I guess the convenience is worth it. I like the name base_test_* but this is fine too

Reply:
I also like the names you suggested. I renamed the scripts.

Reply:
I actually figured out another solution, which is to have these base scripts as [a string template in XLML repo](https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/283/files#diff-495d4687b5bff3f6b5a39a3b16c59222e8f638729e635569a8e7d8ea93d351f9R45). If it looks good, I will remove the scripts from this PR. The change in `gpu_multi_process_run.sh` is still required.

Reply:
Ya I like this template in XLML, defined as close as possible to where it is used